bokehjs/test/document.coffee

```diff
@@ -550,11 +550,7 @@ describe "Document", ->
     expect(out).to.be.equal "Bokeh: JS/Python version mismatch\nBokeh: Library versions: JS (#{js_version})  /  Python (0.0.1)\n"
 
     parsed['version'] = "#{js_version}-foo"
-    out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
-    expect(out).to.be.equal ""
-
-    parsed['version'] = "#{js_version}rc123"
-    out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
+    out = stderrTrap -> Document.from_json_string(JSON.stringify(parsed))
```

Comment:
So the test I added was redundant, but the reason it was failing is because it was capturing stdout instead of stderr. I fixed the problem with the previous test, which was capturing stdout. If the test failed you wouldn't see the error.